### PR TITLE
Add viewer and auditor roles for strides

### PR DESCRIPTION
**Pull request by zaro0508**
Setup a Jumpcloud SAML app to allow jumpcloud users
to login to the strides account with the
[viewer and auditor job function roles](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_job-functions.html).

### CODE REVIEW

**Review by ahayden — CHANGES_REQUESTED** [review]
Minor request... please fix typo

**Comment by zaro0508** [comment]
where's the typo @ahayden ?

File: sceptre/scipool/config/strides/jumpcloud-idp.yaml [hunk 1]
```diff
@@ -7,5 +7,7 @@ stack_tags:
 parameters:
   StridesAdminMetadata: !file_contents idp/jumpcloud/strides-admin.xml
   StridesCommunityMgrMetadata: !file_contents idp/jumpcloud/strides-communitymgr.xml
+  StridesViewernMetadata: !file_contents idp/jumpcloud/strides-viewer.xml
```
Comment (ahayden) [inline sceptre/scipool/config/strides/jumpcloud-idp.yaml:10]: This line has a typo @zaro0508 .. `StridesViewernMetdata`

Comment (zaro0508) [reply sceptre/scipool/config/strides/jumpcloud-idp.yaml]: fixed

**Review by ahayden — APPROVED** [review]
ty!

**Pull request merged**